flask_apscheduler/scheduler.py

```diff
@@ -89,6 +89,10 @@ def start(self, paused=False):
         Start the scheduler.
         :param bool paused: if True, don't start job processing until resume is called.
         """
+
+        if is_debug_mode() and not is_werkzeug_reloader_process():
```

Comment:
Can you replace `is_debug_mode` by `flask.helpers.get_debug_flag` and `is_werkzeug_reloader_process` by `werkzeug.serving.is_running_from_reloader()`?

Reply:
ok ,I will change it